ui/components/multichain/token-list-item/price/percentage-and-amount-change/percentage-and-amount-change.tsx

```diff
@@ -143,5 +147,7 @@ export const PercentageAndAmountChange = ({
     }
   }
 
+  console.log(formattedValue, formattedValuePrice);
```

Comment:
Will eventually need removal.

Reply:
WHOOPS ty for catching this!

Reply:
Removed, thanks for catching this - I was using it to assist writing tests.

Reply:
I'll sweep through to see there are no other logs are added.